### ISSUE TITLE
[v3.0.2.0 - Concept] - jQuery Alert function

### ISSUE THREAD

**Issue by straightlight** [opencart/opencart#6039]
On the forum, I have posted a ... 'short' solution on how and where to disable the alert function from the TWIG files: https://forum.opencart.com/viewtopic.php?f=24&t=191728#p690898

**Comment by danielkerr** [comment]
ok i will start fixing all these as soon as i have finished updating the backup system. keep; them, coming please!

**Comment by mhcwebdesign** [comment]
@straightlight: I don't agree with your proposal. Merely commenting out the Javascript alert calls isn't enough. If there are failed Ajax calls then this is a serious condition, e.g. it could be a sign that the server is down. and for these circumstances the end user should see a clear error message on his screen!

**Comment by danielkerr** [comment]
It would be better if we could Ajax the error to its own error log

**Comment by straightlight** [comment]
`It would be better if we could Ajax the error to its own error log`

That would be one option, yes. Also the use of the already implemented: 

`alert(thrownError + "\r\n" + xhr.statusText + "\r\n" + xhr.responseText);`
